push-notifications/android/src/main/AndroidManifest.xml

```diff
@@ -8,4 +8,5 @@
             </intent-filter>
         </service>
     </application>
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
```suggestion
```
For push notifications the plugin doesn't need to include the permission because firebase-messaging is already adding it